App/System/Controller.php

```diff
@@ -132,55 +128,51 @@ private function getSkippedData($pathToSkipFile)
 
     /**
      * Create series folder if not exists.
-     *
-     * @param $serieSlug
      */
-    public function createSerieFolderIfNotExists($serieSlug)
+    public function createSerieFolderIfNotExists(string $serieSlug): void
     {
-        $this->createFolderIfNotExists(SERIES_FOLDER . '/' . $serieSlug);
+        $this->createFolderIfNotExists(SERIES_FOLDER.'/'.$serieSlug);
     }
 
     /**
      * Create folder if not exists.
-     *
-     * @param $folder
      */
-    public function createFolderIfNotExists($folder)
+    public function createFolderIfNotExists($folder): void
     {
         if ($this->system->has($folder) === false) {
-            $this->system->createDir($folder);
+            $this->system->createDirectory($folder);
         }
     }
 
     /**
      * Create cache file
      *
-     * @param array $data
+     *
      * @throws \League\Flysystem\FileExistsException
      * @throws \League\Flysystem\FileNotFoundException
      */
-    public function setCache($data)
+    public function setCache(array $data): void
     {
         $file = 'cache.php';
 
         if ($this->system->has($file)) {
             $this->system->delete($file);
         }
 
-        $this->system->write($file, '<?php return ' . var_export($data, true) . ';' . PHP_EOL);
+        $this->system->write($file, '<?php return '.var_export($data, true).';'.PHP_EOL);
     }
 
     /**
      * Get cached items
      *
      * @return array
      */
-    public function getCache()
+    public function getCache(): array|string
     {
         $file = 'cache.php';
 
         return $this->system->has($file)
-            ? require $this->system->getAdapter()->getPathPrefix() . $file
+            ? ''.$file
```

Comment:
:warning:

Reply:
getPathPrefix was removed from the flysystem (https://github.com/thephpleague/flysystem/issues/1436) but I wasn't sure what this was trying to do so just left the default that the flysystem uses for the path prefix.